lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -583,6 +584,21 @@ module.exports = class MethodMappings {
           return value;
         },
 
+        async isElementPresent(webElementOrId) {
+          try {
+            const element = this.getWebElement(webElementOrId);
```

Comment:
The problem here is that if you use VSCode Debugger and come here, you'll notice that `webElementOrId` is actually a `WebElementPromise` (pay attention to 'Promise' here), which means it is not yet certain if the result after the promise is resolved would even exist or not.

So, you may try to resolve that promise first to see the result and then match that result with `WebElement` or `ShadowRoot`.

After doing that, please try to run the command in one of example tests (and by example tests, I mean the tests present in `examples/tests` directory) and see if the command is actually working as expected.

Reply:
Sure I will fix this soon.

Reply:
I have tried to update the method-mappings and got the result as false when element is not present but still I am finding this error. I guess isPresent searches for a time period 5000ms and then give out false if it does not find an element. So to fix this what would be a correct approach? `
  - OTHER ERRORS:
  Error
    Error
   Timed out while waiting for element "By(css selector, input[name="z"])" to be present for 5000 milliseconds.` 
<img width="835" alt="image" src="https://github.com/nightwatchjs/nightwatch/assets/91911367/12166bb8-cfc8-4c48-b67e-2da7f7e11350">